src/client/components/CloseButton/CloseButton.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { Cross } from '../icons/Cross'
+
+const Button = styled.button`
+  width: 1.5rem;
+  height: 1.5rem;
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  border-radius: 50%;
+  border: none;
+  cursor: pointer;
+  background: ${colorsV3.white};
```

Comment:
super nit: consider using `background-color:` if we just want to change the background color and not any other properties like `background-image, background-position` etc